### PR TITLE
Multiple mobile platforms

### PR DESCRIPTION
**Pull request by meandavejustice**
fixes 3672

txp-multi-platform-detail
txp-multi-platform

### CODE REVIEW

**Comment by meandavejustice** [comment]
@flodolo Can I get your approval on this one?

**Review by flodolo — APPROVED** [review]
Yup, missed this one.

File: frontend/src/app/containers/ExperimentPage/ExperimentControls.js
Comment (lmorchard) [inline frontend/src/app/containers/ExperimentPage/ExperimentControls.js]: Seems like there's a little bit of redundant code here covering the ios, android, ios & android cases. You might be able to get away with just something like this:
```javascript
const mobileControls = () => (
  <Fragment>
    {(platforms.includes("ios")) &&
      <MobileTriggerIOSButton {...{ doShowMobileAppDialog, color: "default" }} />}
    {(platforms.includes("android")) &&
      <MobileTriggerAndroidButton {...{ doShowMobileAppDialog, color: "default" }} />}
  </Fragment>
);
```
(Didn't actually try this, so hopefully it's not a syntax error! 🤷‍♂️ )

File: frontend/src/app/components/MobileDialog/index.js
Comment (lmorchard) [inline frontend/src/app/components/MobileDialog/index.js]: Just a nit: Might help to make this a little more readable if this got broken up over a couple lines. No biggie.

**Review by lmorchard — APPROVED** [review]
Just a couple of small nitpicks, but nothing I'd block merging over.

**Comment by meandavejustice** [comment]
Addressed comments, thanks @lmorchard

**Pull request merged**
